src/main/scala/io/iohk/ethereum/consensus/pow/EthashMiner.scala

```diff
@@ -0,0 +1,205 @@
+package io.iohk.ethereum.consensus
```

Comment:
a note on commit organisation: this would be way more readable if the package change was done in a previous commit and the code changes were done in this one. this way the actual code change would be highlighted and not the whole class.

Reply:
👍